views/partials/head.hbs

```diff
@@ -53,7 +53,7 @@
     ga('create', 'UA-160365006-1', 'auto');
     ga('send', 'pageview');
   </script>
-  {{#eq currentLocale "en-US"}}
-  <script async src="https://w.appzi.io/bootstrap/bundle.js?token=JW19E"></script>
-  {{/eq}}
+  {{!-- {{#eq currentLocale "en-US"}}
```

Comment:
I left this intentionally here as we will be using it for the new feedback mechanism